be/src/vec/exec/scan/vscan_node.cpp

```diff
@@ -1329,11 +1317,15 @@ VScanNode::PushDownType VScanNode::_should_push_down_in_predicate(VInPredicate*
 Status VScanNode::_prepare_scanners(const int query_parallel_instance_num) {
     std::list<VScannerSPtr> scanners;
     RETURN_IF_ERROR(_init_scanners(&scanners));
```

Comment:
better change init_scanner function, call `RETURN_IF_ERROR(_init_scanners(&_scanners))`
remove the useless std::list<VScannerSPtr> scanners;